suite-common/connect-popup/src/methodHooks/ethereumSignTransaction.ts

```diff
@@ -0,0 +1,111 @@
+import { Bip43Path, getNetworkByEvmChainId } from '@suite-common/wallet-config';
+import {
+    accountsActions,
+    selectAccounts,
+    selectSelectedDevice,
+    sendFormActions,
+} from '@suite-common/wallet-core';
+import TrezorConnect from '@trezor/connect';
+import type { EthereumSignTransaction } from '@trezor/connect';
+import { getSerializedPath, validatePath } from '@trezor/connect/src/utils/pathUtils';
```

Comment:
it is probably ok to import from @trezor/connect here. still maybe these utils should be one day factored out to some common package

Reply:
It's not totally ideal but I believe this specific file shouldn't be a problem